app/src/androidTest/java/com/team7/cmput301/android/theirisproject/DeleteProblemActivityTest.java

```diff
@@ -14,7 +14,7 @@
 import com.team7.cmput301.android.theirisproject.task.Callback;
 
 /**
- * Tests for EditPatientProfileActivity
+ * Tests for EditProfileActivity
```

Comment:
Should be "Tests for DeleteProblemActivity", although that was probably originally my mistake